qiskit/transpiler/passes/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 """Module containing transpiler pass."""
 
+from qiskit.transpiler.passes.unroller import Unroller
```

Comment:
for consistency with other import styles in this file
```suggestion
from .unroller import Unroller
```